packages/clients/meldemichel/src/plugins/AfmButton/index.ts

```diff
@@ -0,0 +1,12 @@
+import Vue from 'vue'
+import { PluginOptions } from '@polar/lib-custom-types'
+import AfmButton from './AfmButton.vue'
+import language from './language'
+
+export default (options: PluginOptions) => (instance: Vue) =>
+  instance.$store.dispatch('addComponent', {
+    name: 'scale',
```

Comment:
```suggestion
    name: 'afmButton',
```
Or `meldemichelAfmButton`, if you insist :D

Reply:
https://github.com/Dataport/polar/pull/23/commits/3d3a27cac7ae413f174e29ad0b28413efe4e6f35